projects/openapi-utilities/src/diff/diff.ts

```diff
@@ -122,17 +129,40 @@ export function diff(before: any, after: any): ObjectDiff[] {
           afterPath,
         });
       }
-    } else if (!Array.isArray(before.value) && !Array.isArray(after.value)) {
+    } else if (
+      !Array.isArray(before.value) &&
+      !Array.isArray(after.value) &&
+      Object.prototype.toString.call(before.value) &&
```

Comment:
I think this was running before on primitives, but `Object.keys(...)` was returning `[]`